Sound/Tidal/Pattern.hs

```diff
@@ -842,6 +842,7 @@ d1 $ sound (samples "xx(3,8)" (tom $ choose ["a", "e", "g", "c"]))
 plays a melody randomly choosing one of the four notes \"a\", \"e\", \"g\", \"c\".
 -}
 choose :: [a] -> Pattern a
+choose [] =  throw (ErrorCall "Empty list. Nothing to choose from.")
```

Comment:
This is the actual line I've added. Don't know why the others lines appear in this PR.